pkgs/tools/text/catdoc/default.nix

```
@@ -16,6 +16,12 @@ stdenv.mkDerivation rec {
     })
   ];
 
+  # Remove INSTALL file on Darwin to avoid `make` misinterpret it as up-to-date
+  # target on case-insensitive filesystems.
+  preInstall = lib.optionalString stdenv.isDarwin ''
```

I don't think it has any disadvantages to do this everywhere. This way we avoid bitrot.

```suggestion
  preInstall = ''
```